code/modules/surgery/bodyparts/parts.dm

```diff
@@ -305,6 +306,7 @@
 	unarmed_damage_low = 2
 	unarmed_damage_high = 15
 	unarmed_stun_threshold = 10
+	icon_state = "default_human_r_leg"
```

Comment:
this isnt why the linter is unhappy. remove this and in `limbsanity.dm`, change `subtypesof(/obj/item/bodypart)` to `subtypesof(/obj/item/bodypart/) - list(/obj/item/bodypart/arm, /obj/item/bodypart/leg)`

Since these shouldnt exist in game we dont care if they have invalid appearances

Reply:
gotcha